src/writer/command_line.py

```diff
@@ -73,6 +73,23 @@ def hello(port: Optional[int], host: str, enable_remote_edit):
        enable_server_setup=False)
 
 
+@main.command()
+@click.option('--api-key',
+    default=lambda: os.environ.get("WRITER_API_KEY", None),
+    allow_from_autoenv=True,
+    show_envvar=True,
+    envvar='WRITER_API_KEY',
+    prompt="Enter your API key",
+    hide_input=True, help="Writer API key"
+)
+@click.option('--env', '-e', multiple=True, default=[], help="Environment to deploy the app to")
```

Comment:
Maybe `Environments` as it can be multiple

```suggestion
@click.option('--env', '-e', multiple=True, default=[], help="Environments to deploy the app to")
```

Reply:
Aren't these environment variables?